### PR TITLE
unused files deleted + Map component functional + body prettier OK

### PR DESCRIPTION
**Pull request by paulinedero**

### CODE REVIEW

File: src/Components/Body/Body.css [hunk 1]
```diff
@@ -80,6 +80,7 @@
     padding: 5px;
     border-radius: 5px;
     font-family: 'Roboto', sans-serif;
+    font-size: 12pt;
```
Comment (MP-C) [inline src/Components/Body/Body.css:83]: Ok

**Review by MP-C — APPROVED** [review]
Correct for me

**Review by xdewez — APPROVED** [review]
Valider modifs Pailine

**Review by xdewez — APPROVED** [review]
Valider

**Review by xdewez — APPROVED** [review]
Valider 2

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "@testing-library/react": "^11.2.6",
     "@testing-library/user-event": "^12.8.3",
     "axios": "^0.21.1",
+    "country-reverse-geocoding": "^0.2.2",
```
Comment (MP-C) [inline package.json:10]: Good

File: src/Components/Body/Body.css (hunk 1 above)
Comment (MP-C) [inline src/Components/Body/Body.css:83]: good

File: src/Components/Body/Body.js [hunk 3]
```diff
@@ -1,10 +1,9 @@
 import axios from 'axios';
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
```
Comment (MP-C) [inline src/Components/Body/Body.js:2]: good

File: src/Components/Body/Body.js [hunk 4]
```diff
@@ -72,28 +84,30 @@ export default function Body() {
...
             <div id="column-right" className="map">
                 <div id="search-result">
-                    <Map
-                        lat = {lat}
-                        lng=  {lng}
-                    />
+                    <Map lat={lat} lng={lng} />
                 </div>
             </div>
         </div>
```
Comment (MP-C) [inline src/Components/Body/Body.js:113]: good

**Review by MP-C — APPROVED** [review]
Impec

**Review by xdewez — APPROVED** [review]
Valid

**Review by MP-C — APPROVED** [review]

**Pull request closed without merging**